docs/ja/docs/tutorial/cors.md

```diff
@@ -0,0 +1,84 @@
+# CORS (オリジン間リソース共有)
+
+<a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS" class="external-link" target="_blank">CORSまたは「オリジン間リソース共有」</a> は、ブラウザで実行されているフロントエンドにバックエンドと通信するJavaScriptコードがあり、そのバックエンドがフロントエンドとは異なる「オリジン」にある状況を指します。
+
+## オリジン
+
+オリジンはプロトコル (`http`、`https`) とドメイン (`myapp.com`、`localhost`、`localhost.tiangolo.com`) とポート (`80`、`443`、`8080`) の組み合わせです。
+
+したがって、以下はすべて異なるオリジンです:
+
+* `http://localhost`
+* `https://localhost`
+* `http://localhost:8080`
+
+すべて `localhost` であっても、異なるプロトコルやポートを使用するので、異なる「オリジン」です。
+
+## ステップ
+
+そして、ブラウザ上で実行されているフロントエンド (`http://localhost:8080`) があり、そのJavaScriptが `http://localhost` で実行されているバックエンドと通信するとします。(ポートを指定していないので、ブラウザはデフォルトの`80`ポートを使用します)
+
+次に、ブラウザはHTTPの `OPTIONS` リクエストをバックエンドに送信します。そして、バックエンドがこの異なるオリジン (`http://localhost:8080`) からの通信を許可する適切なヘッダーを送信すると、ブラウザはフロントエンドのJavaScriptにバックエンドへのリクエストを送信させます。
+
+これを実現するには、バックエンドに「許可されたオリジン (allowed origins)」のリストがなければなりません。
+
+この場合、フロントエンドを正しく機能させるには、そのリストに `http://localhost:8080` を含める必要があります。
+
+## ワイルドカード
+
+リストを `"*"` (ワイルドカード) と宣言して、すべてを許可することもできます。
+
+ただし、Bearer Tokenで使用されるような認証ヘッダーやCookieなどのクレデンシャル情報に関するものを除いて、特定の種類の通信のみが許可されます。
```

Comment:
Translation of "Cookie" should be "Cookie" or "クッキー" ??
I found some of the pages used "クッキー", so I wanted to be consistent with wording.
(https://github.com/tiangolo/fastapi/pull/1933/files)

I prefer "Cookie" than "クッキー", because cookie is common technical term.

Reply:
hmmm... let us use `cookie` because it is used in browser settings.
I may use both of them in other PR's. I'll check it.

Reply:
can you list it in main issue?
let us discuss if you find other words.